tests/fixtures/testrepo/downstream/package_manifest.yaml

```diff
@@ -1,12 +1,15 @@
 ---
 packages:
   vars:
-    diff_brew_package_skip: false
+    diff_package_skip: false
+    scl: tfm
     build_package_koji_command: 'brew'
-    satellite_version: '6.3.0'
+    obaltest_version: '6.3.0'
```

Comment:
LOL, thanks for spotting this. This line can go now, after your refactoring. :boom: